extensions/integration-tests/package.json

```diff
@@ -49,15 +49,20 @@
     "postinstall": "node ./node_modules/vscode/bin/install"
   },
   "devDependencies": {
-    "@types/node": "7.0.43",
     "@types/chai": "3.4.34",
+    "@types/node": "7.0.43",
+    "chai": "3.5.0",
+    "class-validator": "^0.9.1",
     "mocha-junit-reporter": "^1.17.0",
     "mocha-multi-reporters": "^1.1.7",
-    "vscode": "1.1.5",
-    "chai": "3.5.0"
+    "azdata-test": "d:/src/azdata-test/dist",
+    "ts-node": "^8.1.0",
```

Comment:
I'm surprised this actually works - I was under the impression that dependencies were semver package versions only. Regardless you can't have hardcoded paths like this - not everyone is going to have the same filesystem layout (even if you ignore the non-Windows platforms)

Reply:
npm and yarn support various formats, local folder, remote git repos, or semver.

Reply:
So it does - probably could have just read the documentation.

https://docs.npmjs.com/files/package.json#dependencies

Reply:
I have just re-ordered these. These were pre-existing so not changing these at this time.

---
In reply to: [287893608](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r287893608) [](ancestors = 287893608)